pkg/builtin/unary/ceil.go

```diff
@@ -0,0 +1,238 @@
+package unary
+
+import (
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/container/nulls"
```

Comment:
Split stdlib and self contained lib into two chunks?

Reply:
Sorry, this is my mistake, it has been corrected